api/structure/schema.py

```diff
@@ -44,6 +45,7 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, **kwargs):
+        validate_structure_id(check_unique_structure_id=True, **kwargs)
```

Comment:
Great work @jesseinit your PR works as described, however, do you need this argument `check_unique_structure_id=True`,?  without it and the flag for it in your validator, the PR still works as expected, kindly elaborate on this.

Reply:
Without the check, the PR wouldn't work as expected, It'll throw an integrity error. The check is only making sure that the structures being created doesn't already exist in the database.